<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWidget</class>
 <widget class="QWidget" name="WelcomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="welcomeLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Welcome to Lokalize!&lt;/p&gt;&lt;p&gt;Drop a translation file here to start editing it.&lt;/p&gt;&lt;p&gt;You can help translating KDE into your mother language by going to &lt;a href=&quot;http://l10n.kde.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;KDE localization site&lt;/span&gt;&lt;/a&gt; and contacting appropriate language team.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ln">
     <property name="text">
      <string>Your name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="authorNameEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lr">
     <property name="text">
      <string>Your role (XLIFF only):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="roleCombo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ls">
     <property name="text">
      <string>Default source language:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lt">
     <property name="text">
      <string>Default target language:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="sourceLangCombo"/>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="targetLangCombo"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lg">
     <property name="text">
      <string>Glossary file:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="glossaryPathEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
